api/src/modules/sourcing-record-groups/sourcing-record-groups.entity.ts

```diff
@@ -0,0 +1,40 @@
+import {
```

Comment:
File from https://github.com/Vizzuality/landgriffon/pull/46 PR (**And duplicated entity?**)